onboarding.md

```diff
@@ -0,0 +1,28 @@
+# Onboarding
+
+All participants in the OpenJS Foundation projects and groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md).
+There are further expectations for members who represent the Standards Working Group (hereby called representatives).
+
+It is understood that representatives will be a part of individual or their company specific projects. But while representing
+the Standards Working Group, they should take decisions/discuss/promote in the interest of Standards Group. 
+
+If there is a conflict (due to personal or company specific projects), then the representative should be explicit 
+about the hat they are wearing while deciding/discussing/promoting the project.
+
+Since many members are often a part of one or more projects in the foundation, feel free to discuss it with the members 
+if there is a potential conflict while representing. The representative can either raise an issue and discuss them during 
+the Standards Team meeting.
+
+If a representative has an objection or dissent they should express it as early as possible to ensure 
+there is ample time to discuss and reach consensus before representing Standards Working Group.
+
+Representatives must also conduct themselves in a professional and respectful manner. Some general guidelines include:
+
+* When in doubt always wear the least possible hat.
+* Aim to remediate first and then discuss. If other members of the team express concerns about actions, acknowledge their concerns by stopping the actions in question and then discuss within the team to come to a common agreement.
+* Treat all community members with respect, consideration, and highest standards of ethical conduct.
+* Build trust by keeping your promises.
+* Be the model of accountability and leadership. Provide the example of ownership and stewardship that everyone can follow to success.
+* Commit to ongoing development and learning best practices for governing.
```

Comment:
Is their a primer for these best practices that you've found useful?

Reply:
I took this from the CPC's Membership Expectations https://github.com/openjs-foundation/cross-project-council/blob/master/MEMBER_EXPECTATIONS.md